### PR TITLE
Clipboard+

### PR DESCRIPTION
**Pull request by brachy84**
- reworked all clipboard textures to make it look more like a clipboard
- fix a issue where multiple textfields can be selected
- fix place behaviour. Before it was facing player look dir, now it is block face placed on
- when the neighbour changed and the block it is placed on becomes invalid the clipboard breakes

old
grafik

new
grafik

### CODE REVIEW

File: src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java [hunk 1]
```diff
@@ -195,25 +208,44 @@ public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fac
...
-                this.onRemoval();
+            Block.spawnAsEntity(getWorld(), this.getPos(), drops.get(0));
+            this.dropAllCovers();
+            this.onRemoval();
 
-                world.removeTileEntity(pos);
-                world.setBlockState(pos, Blocks.AIR.getDefaultState(), 3);
-                this.scheduleRenderUpdate();
+            world.setBlockState(pos, Blocks.AIR.getDefaultState(), 3);
+            this.scheduleRenderUpdate();
```
Comment (Rongmario) [inline src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java:229]: setBlockState with flag 3 should sync to client and ask for a render update, would `scheduleRenderUpdate` be needed?

File: src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java [hunk 2]
```diff
@@ -195,25 +208,44 @@ public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fac
...
 
-                NonNullList<ItemStack> drops = NonNullList.create();
-                getDrops(drops, playerIn);
+            NonNullList<ItemStack> drops = NonNullList.create();
+            getDrops(drops, player);
 
-                Block.spawnAsEntity(playerIn.world, this.getPos(), drops.get(0));
-                this.dropAllCovers();
-                this.onRemoval();
+            Block.spawnAsEntity(getWorld(), this.getPos(), drops.get(0));
```
Comment (Rongmario) [inline src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java:224]: Use the `pos`?

**Review by Rongmario — CHANGES_REQUESTED** [review]

**Comment by ChromaPIE** [comment]
Had a little test with #645 also merged into my fork. The clipboard follows the UI color setting in the config in-world, but the overlay GUI isn't applying the color setting. It should not be affected by the color setting, my pov.

image

image

**Comment by brachy84** [comment]
Will fix after this or the other pr is merged

**Comment by DStrand1** [comment]
> Had a little test with https://github.com/GregTechCEu/GregTech/pull/645 also merged into my fork. The clipboard follows the UI color setting in the config in-world, but the overlay GUI isn't applying the color setting. It should not be affected by the color setting, my pov.

I will update this in #645 after this merges

**Review by Rongmario — APPROVED** [review]

**Pull request merged**